charts/openstack-cloud-controller-manager/values.yaml

```diff
@@ -46,6 +46,8 @@ livenessProbe: {}
 # Set readinessProbe in the same way like livenessProbe
 readinessProbe: {}
 
+# set dnsPolicy to 'ClusterFirstWithHostNet' if the cluster is deployed behind a corporate reverse proxy
+dnsPolicy: default
```

Comment:
Why didn't you set default value to `ClusterFirstWithHostNet`?

Reply:
to leave the default option and leave the user to modify it ONLY if needed :)